src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.cs

```diff
@@ -99,15 +99,13 @@ private static CalendarData CreateInvariant()
         //
         // Get a bunch of data for a calendar
         //
-        internal CalendarData(string localeName, CalendarId calendarId, bool bUseUserOverrides)
+        internal CalendarData(string localeName, CalendarId calendarId, bool bUseUserOverrides, bool isUserDefaultLocale)
```

Comment:
>bool bUseUserOverrides, bool isUserDefaultLocale [](start = 72, length = 48)

can we combine these 2 flags. I mean sending bUseUserOverrides = true only when it is user default locale?

Reply:
Yes we can do that, that would affect NLS though but seems reasonable to only ask for user overrides when we're in the default culture.